core/repository/consensus/merkle_transaction_repository.hpp

```diff
@@ -9,7 +9,7 @@
 
 namespace merkle_transaction_repository {
 bool commit(std::string const hash, std::unique_ptr<consensus_event::ConsensusEvent> const tx);
-bool commit(std::string, std::unique_ptr<consensus_event::ConsensusEvent> event);
+bool commit(std::string, std::unique_ptr<consensus_event::ConsensusEvent> const event);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'event' should be passed by reference.](https://www.codacy.com/app/soramitsu/iroha/file/4256066340/issues/source?bid=3712501&fileBranchId=3831026#l12)